From 39417c1106240282eedaa7872233d593590df3ca Mon Sep 17 00:00:00 2001
Message-ID: <39417c1106240282eedaa7872233d593590df3ca.1701297273.git.sharlatanus@gmail.com>
In-Reply-To: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
References: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 28 Oct 2023 03:56:34 +0100
Subject: [PATCH v3 20/66] gnu: go-github-com-jcmturner-goidentity-v6: Adjust
 inputs.

* gnu/packages/golang-web.scm (go-github-com-jcmturner-goidentity-v6):
[propagated-inputs]: Move go-github-com-stretchr-testify from here ...
[native-inputs]: ... to here.

Change-Id: I1e7bd72b3bf1c0879426a6a4c5b3b1de1d316069
---
 gnu/packages/golang-web.scm | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 751eb41b58..8ced6a4c6f 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -330,8 +330,10 @@ (define-public go-github-com-jcmturner-goidentity-v6
     (build-system go-build-system)
     (arguments
      '(#:import-path "github.com/jcmturner/goidentity/v6"))
+    (native-inputs
+     (list go-github-com-stretchr-testify))
     (propagated-inputs
-     (list go-github-com-stretchr-testify go-github-com-hashicorp-go-uuid))
+     (list go-github-com-hashicorp-go-uuid))
     (home-page "https://github.com/jcmturner/goidentity")
     (synopsis "Hold authenticated identities and their attributes")
     (description "This package provides a standard interface for holding
-- 
2.41.0

